lib/Elevate/Components/JetBackup.pm

```diff
@@ -84,24 +91,12 @@ sub post_distro_upgrade ($self) {
 
 sub check ($self) {
 
-    $self->_blocker_jetbackup_is_supported();
     $self->_blocker_old_jetbackup;
 
     return;
 }
 
-# Support for JetBackup on Ubuntu is scheduled via RE-668
-# For now, we block to reduce scope for the initial release
-sub _blocker_jetbackup_is_supported ($self) {
-    return unless Cpanel::Pkgr::is_installed('jetbackup');
-    return if Elevate::OS::supports_jetbackup();
-
-    my $name = Elevate::OS::default_upgrade_to();
-    return $self->has_blocker( <<~"END" );
-    ELevate does not currently support JetBackup for upgrades of $name.
-    Support for JetBackup on $name will be added in a future version of ELevate.
-    END
-}
+sub _blocker_jetbackup_is_supported { return undef }
```

Comment:
TRIVIAL: Does anything call this / could it just go away?

Reply:
this will come up again on 8->9....

Reply:
Only the test for it